### PR TITLE
Support search by multiple trace IDs

### PR DESCRIPTION
**Pull request by yurishkuro**

### CODE REVIEW

File: cmd/query/app/handler.go [hunk 1]
```diff
@@ -181,9 +182,22 @@ func (aH *APIHandler) search(w http.ResponseWriter, r *http.Request) {
...
 	}
-	tracesFromStorage, err := aH.spanReader.FindTraces(tQuery)
-	if aH.handleError(w, err, http.StatusInternalServerError) {
-		return
+
+	var tracesFromStorage []*model.Trace
+	if len(tQuery.traceIDs) > 0 {
+		tracesFromStorage, err = aH.tracesByIDs(tQuery.traceIDs)
+		if err == spanstore.ErrTraceNotFound {
+			aH.handleError(w, err, http.StatusBadRequest)
```
Comment (black-adder) [inline cmd/query/app/handler.go]: return not found 404 instead

File: cmd/query/app/handler.go [hunk 2]
```diff
@@ -203,6 +217,18 @@ func (aH *APIHandler) search(w http.ResponseWriter, r *http.Request) {
 	aH.writeJSON(w, &structuredRes)
 }
 
+func (aH *APIHandler) tracesByIDs(traceIDs []model.TraceID) ([]*model.Trace, error) {
+	retMe := make([]*model.Trace, 0, len(traceIDs))
+	for i := range traceIDs {
```
Comment (black-adder) [inline cmd/query/app/handler.go]: nit: why not `for _, traceID := range traceIDs`

Comment (yurishkuro) [inline cmd/query/app/handler.go]: no reason

File: cmd/query/app/handler.go (hunk 1 above)
Comment (yurishkuro) [inline cmd/query/app/handler.go]: good call

**Pull request merged**